data.sql

```diff
@@ -1,8 +1,22 @@
 /* Populate database with sample data. */
 
+-- Day I
+
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES
 ('Agumon', '2020-02-03', 0, TRUE, 10.23),
 ('Gabumon', '2018-11-15', 2, TRUE, 8),
 ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
 ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+-- Day II
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '2020-02-08', 0, FALSE, 11),
+('Plantmon', '2021-11-15', 2, TRUE, 5.7),
+('Squirtle', '1993-04-02', 3, FALSE, 12.13),
+('Angemon', '2005-06-12', 1, TRUE, 45),
```

Comment:
- [ ] I know it looks like a mistake, but here, these three animals are supposed to have negative weight, so please change it from 5.7, 12.13, and 45 to -5.7, -12.13, and -45.

Reply:
Okay. Although then it looks more like science fiction than a DB :). 
By the way, if so, then Charmander must also have a negative weight.

Reply:
I know what you mean, but you will understand the reason for this soon 😎